keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/TestController.kt

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.yaml.APPLICATION_YAML_VALUE
+import com.netflix.spinnaker.kork.exceptions.SystemException
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/test"])
+class TestController {
+
+  @GetMapping(
+    path = ["/error"],
+    produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
+  )
+  fun throwAnException(): String {
```

Comment:
It maybe will make sense it should return `Map<String, Any>` and not `string`, like other controllers in keel that throws an exception?

Reply:
Agreed!

Reply:
Will do.

Reply:
Addressed in 2914d241